### PR TITLE
update(HTML): web/html/element/input/checkbox

### PR DESCRIPTION
**Pull request by undead404**
Оригінальний вміст: [&lt;input type="checkbox">@MDN](https://developer.mozilla.org/en-us/docs/Web/HTML/Element/input/checkbox), [сирці &lt;input type="checkbox">@GitHub](https://github.com/mdn/content/blob/main/files/en-us/web/html/element/input/checkbox/index.md)

Нові зміни:
- [mdn/content@e9b1ea6](https://github.com/mdn/content/commit/e9b1ea6ad21c6186ab07d8de4f8b967d8001eef5)
- [mdn/content@1ea507b](https://github.com/mdn/content/commit/1ea507bda02c3a33287c4567955d29071a572b89)
- [mdn/content@0053e22](https://github.com/mdn/content/commit/0053e22667809069b5a62c3de80122cdb65d611c)
- [mdn/content@fa8a44b](https://github.com/mdn/content/commit/fa8a44b8bff24a4032181c4fd155c459c0dc9161)

### CODE REVIEW

File: files/uk/web/html/element/input/checkbox/index.md [hunk 1]
```diff
@@ -46,17 +46,17 @@ browser-compat: html.elements.input.type_checkbox
 
 Якщо атрибут `value` опущений, то усталеним значенням поля буде `on`, тож подані дані в такому випадку будуть `subscribe=on`.
 
-> **Примітка:** Якщо поле для галочки пусте при подачі форми, то жодне значення, що відображає відсутність галочки, не буде подано на сервер (наприклад, `value=unchecked`); значення не подається на сервер узагалі. При потребі подати усталене значення поля для галочки, коли воно пусте, можна включити у форму {{HTMLElement("input/hidden", '&lt;input type="hidden"&gt;')}} з такими самими `name` і `value`, згенероване, мабуть, JavaScript.
+> **Примітка:** Якщо поле для галочки пусте при подачі форми, то на сервер не подаються ані назва, ані значення. Не існує суто HTML метода подати непозначений стан поля для галочки (виду `value=unchecked`). При потребі подати усталене значення поля, коли воно пусте, можна включити JavaScript для створення всередині форми {{HTMLElement("input/hidden", '&lt;input type="hidden"&gt;')}} зі значенням, що позначає пустий стан.
```
Comment (ibulik) [inline files/uk/web/html/element/input/checkbox/index.md:49]:
```suggestion
> **Примітка:** Якщо поле для галочки пусте при подачі форми, то на сервер не подаються ані назва, ані значення. Не існує суто HTML метода для подавання непозначеного стану поля для галочки (виду `value=unchecked`). При потребі подати усталене значення поля, коли воно пусте, можна використати JavaScript для створення всередині форми {{HTMLElement("input/hidden", '&lt;input type="hidden"&gt;')}} зі значенням, що позначає пустий стан.
```

File: files/uk/web/html/element/input/checkbox/index.md [hunk 2]
```diff
@@ -46,17 +46,17 @@ browser-compat: html.elements.input.type_checkbox
...
 
 - `checked` (перевірений)
 
-  - : Булів атрибут, що вказує, чи має поле галочку спочатку (при завантаженні сторінки). Воно _не_ вказує, чи є поле порожнім наразі: якщо стан поля зміниться, цей атрибут вмісту не відобразить змін. (Оновиться лише IDL атрибут {{domxref("HTMLInputElement")}}'а `checked`ʼ.)
-    > **Примітка:** На відміну від інших полів введення, значення поля для галочки включається у дані подання лише якщо поле позначено (`checked`). Якщо воно таким є, то значення атрибута `value` стає значенням поля у формі.
-    > На відміну від інших браузерів, Firefox усталено [зберігає динамічний стан галочки (англ.)](https://stackoverflow.com/questions/5985839/bug-with-firefox-disabled-attribute-of-input-not-resetting-when-refreshing) поля між завантаженнями сторінки. Використовуйте для контролю цього функціоналу атрибут {{htmlattrxref("autocomplete","input")}}.
+  - : [Булів](/uk/docs/Glossary/Boolean/HTML) атрибут, що вказує, чи має поле галочку спочатку (при завантаженні сторінки). Воно _не_ вказує, чи є поле порожнім наразі: якщо стан поля зміниться, цей атрибут вмісту не відобразить змін. (Оновиться лише IDL атрибут {{domxref("HTMLInputElement")}}'а `checked`ʼ.)
```
Comment (ibulik) [inline files/uk/web/html/element/input/checkbox/index.md:57]:
```suggestion
  - : [Булів](/uk/docs/Glossary/Boolean/HTML) атрибут вказує, чи має поле галочку спочатку (при завантаженні сторінки). Воно _не_ вказує, чи є поле порожнім наразі: якщо стан поля зміниться, цей атрибут вмісту не відобразить змін. (Оновиться лише IDL атрибут {{domxref("HTMLInputElement")}}'а `checked`ʼ.)
```

Comment (undead404) [reply files/uk/web/html/element/input/checkbox/index.md:57]:
```suggestion
  - : Це [булів](/uk/docs/Glossary/Boolean/HTML) атрибут, що вказує, чи має поле галочку спочатку (при завантаженні сторінки). Воно _не_ вказує, чи є поле порожнім наразі: якщо стан поля зміниться, цей атрибут вмісту не відобразить змін. (Оновиться лише IDL атрибут {{domxref("HTMLInputElement")}}'а `checked`ʼ.)
```
Можливо, краще отак?

Comment (KlashReik) [reply files/uk/web/html/element/input/checkbox/index.md:57]: > Можливо, краще отак?

Давай так

**Review by KlashReik — APPROVED** [review]

**Review by KlashReik — APPROVED** [review]

**Pull request merged**